bower.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "bootstrap-winjs",
-  "version": "0.2.0",
+  "name": "WinStrap",
```

Comment:
We decided to remove the intracaps. Should we update this one as well to be consistent? And there are several others too in this PR.